docs/docs/user-guides/client-instructions.mdx

```diff
@@ -48,22 +57,39 @@ follow the steps below. You will need:
 Admin. If your company is using a Single Sign On provider (like Google or Okta),
 the Firezone portal will prompt you to authenticate via that provider.
 
-### Step 1 - Deactivate VPN session
+### Step 1: Deactivate VPN session
 
 ![WireGuard Deactivate](https://user-images.githubusercontent.com/52545545/156859259-a3d386ce-b304-4caa-96e6-a8e7ca96d098.png)
 
-### Step 2 - Re-authenticate
+### Step 2: Re-authenticate
 
 Visit the URL of your Firezone portal and sign in using credentials provided by your
 network admin. If you are already logged into the portal,
 click the `Reauthenticate` button, then sign in again.
 
 ![re-authenticate](https://user-images.githubusercontent.com/52545545/155812962-9b8688c1-00af-41e4-96c3-8fb52f840aed.gif)
 
-### Step 3 - Activate VPN session
+### Step 3: Activate VPN session
 
 ![Activate Session](https://user-images.githubusercontent.com/52545545/156859636-fde95fc5-5b9c-4697-9108-2f277ed3fbef.png)
 
+## Client app features
+
+### Automatically enable the VPN (on-demand)
+
+:::note
+Requires admin access.
+:::
+
+On-demand can be set to automatically enable the VPN under certain
+network conditions.
+
+On the Android and iOS mobile apps you can automatically
+enable the VPN on Wi-fi and/or Cellular networks.
+
+On the MacOS and Windows desktop apps you can automatically
+enable the VPN on Wi-fi networks and/or Ethernet connections.
+
```

Comment:
We'll get support questions about this. Maybe better to point them to WireGuard's docs or remove this section.

```suggestion
```

Reply:
I don't think WireGuard keeps any documentation on their client apps. I understand we don't want to maintain any docs for a product we don't have control over. Will remove.